pydgin/elf.py

```diff
@@ -528,28 +528,23 @@ def elf_reader( file_obj, is_64bit=False ):
     #section_name = start.partition('\0')[0]
     section_name = start.split('\0', 1)[0]
 
-    # only sections marked as lloc should be written to memory
-
-    if not (shdr.flags & ElfSectionHeader.FLAGS_ALLOC):
-      continue
-
     # Read the section data if it exists
-
-    if section_name not in ['.sbss', '.bss']:
+    if (shdr.type == ElfSectionHeader.TYPE_NULL or
+        section_name == '.comment' or
```

Comment:
It could well be a bug in their format. I'm pretty sure this location contains some data. I'll try reporting this as an issue on one of the Epiphany repos and see what happens.